src/vsr.zig

```diff
@@ -1525,6 +1529,29 @@ test "parse_addresses" {
     }
 }
 
+test "parse_addresses: fuzz" {
+    const test_count = 1024;
+    const len_max = 32;
+    const alphabet = " \t\n,:[]0123456789abcdefgABCDEFGXx";
+
+    const seed = std.crypto.random.int(u64);
```

Comment:
Only other note is whether or not we should use a CPRNG in unit tests -- I think all of our existing unit tests are deterministic, and then we have separate fuzzers to test with real randomness.

But a separate fuzzer feels like overkill in this case, so I'm happy with it as-is.

Reply:
This is copy-pasted from `test "k_way_merge: fuzz"`, so there's a precedent (I didn't add `errdefer` line, as we only test for crashes here, and panics don't call defers).

In general, I am pretty strongly in favor of using real random seed in these kinds of tests --- if the test is only flaky when the code is incorrect, I'd rather have such a test. 

Though, what gives me a pause here is that `zig build test` actually caches the result of the test, defeating the purpose of CPRNG here a bit.